kernel/update_kernel.sh

```diff
@@ -6,26 +6,48 @@
 # Default image to build is the one specified in file configure.ac
 # located at the root of the repository.
 set -x
+KR_URL=https://www.kernel.org/releases.json
+KR_LTS=4.9
+
 AUTHOR=${AUTHOR:-$(git config user.name)}
 AUTHOR_EMAIL=${AUTHOR_EMAIL:-$(git config user.email)}
 
-CC_VERSIONS_FILE="../versions.txt"
```

Comment:
The idea is that packaging/kernel will be the first place where the kernel will be updated and after build test and a PR to packaging  will be propagated to others repos